airgun/views/product.py

```diff
@@ -238,3 +238,20 @@ class ProductSyncPlanView(SyncPlanCreateView):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
+
+
+class ProductManageHttpProxy(BaseLoggedInView):
+    """Represents Http Proxy Management page for Products."""
+    title = Text("//h4[text()='Assign Compliance Policy']")
```

Comment:
will that work? Shouldn't it be
```suggestion
    title = Text("//h4[text()='Http Proxy Management']")
```

Reply:
yep

Reply:
update PR, thanks.